tests/helpers/platform/index.ts

```diff
@@ -124,3 +106,12 @@ export async function setLocation(lat: number, lng: number) {
     altitude: '94.23',
   });
 }
+
+export async function restartApp(urlPath: string) {
+  // this is needed to set the "default" url on web so the DB can be cleared
+  if (isWeb()) {
+    return url(urlPath);
+  }
+
+  return Promise.resolve();
```

Comment:
These promise resolves aren't necessary since the function is marked async already, but it's not a big deal just pointing it out

Reply:
ahh, didn't know that, sorry, will remove them and then we can merge this one

Reply:
@mlynch 
ok, reverted them. Thanks

But out of curiosity , there's also have a function like this

```ts
export async function setLocation(lat: number, lng: number) {
  if (isWeb()) {
    // Not available on web
    return Promise.resolve();
  }

  return driver.setGeoLocation({
    latitude: '' + lat,
    longitude: '' + lat,
    altitude: '94.23',
  });
}
```

Should that one have a `return Promise.resolve();` or a `return` (this is still original code

Reply:
It can just do a `return`, no need to resolve the promise, that happens automatically